core-features/studio/test.sh

```diff
@@ -11,3 +25,26 @@ sleep 3 # Studio takes some time to start up
 yarn jest
 
 kill -9 $PRISMA_PID
+kill -9 $(lsof -t -i:5555)
+
+if [[ -z "$PRISMA_FORCE_NAPI" ]] ; then
+	# If you are supposed to use binaries, check if libraries were downloaded during tests
+	if [[ ! -z "$(ls -1 node_modules/.prisma/client/libquery_engine-* 2>/dev/null)" ]] ; then
+		echo '[1] QE library exists when it should not'
+		exit 1
+	fi
+	if [[ ! -z "$(ls -1 node_modules/prisma/libquery_engine-* 2>/dev/null)" ]] ; then
+		echo '[2] QE library exists when it should not'
+		exit 1
+	fi
+else
+	# If you are supposed to use libraries, check if binaries were downloaded during tests
+	if [[ ! -z "$(ls -1 node_modules/.prisma/client/query-engine* 2>/dev/null)" ]] ; then
+		echo '[1] QE binary exists when it should not'
+		exit 1
+	fi
+	if [[ ! -z "$(ls -1 node_modules/prisma/query-engine* 2>/dev/null)" ]] ; then
+		echo '[2] QE binary exists when it should not'
+		exit 1
+	fi
+fi
```

Comment:
Here are some assertions that should check if anything extra was downloaded.

If you look on line 11, I delete binaries if `PRISMA_FORCE_NAPI` is set, and then check again here if they exist. If they do, it means someone downloaded them, and the test fails.

The same happens if `PRISMA_FORCE_NAPI` is not set, libraries are deleted before `jest`, and checked after.

Reply:
Let me know if all paths I check are sufficient, can obviously add other places we expect to find the binary / library.

Reply:
On Windows the paths should be different for the Node-API library, but I assume that is not being tested here so does not affect things.